src/t_stream.c

```diff
@@ -2734,7 +2734,7 @@ NULL
     }
 }
 
-/* XSETID <stream> <id> [ENTRIESADDED entries_added] [MAXDELETEDID max_deleted_entry_id]
+/* XSETID <key> <id> [ENTRIESADDED entries_added] [MAXDELETEDID max_deleted_entry_id]
```

Comment:
i don't think we need to make an effort to match the argument names

Reply:
roll back according to comments, pls take a look, Thanks